src/scene/constants.js

```diff
@@ -1054,3 +1054,51 @@ export const DITHER_BLUENOISE = 'bluenoise';
  * @category Graphics
  */
 export const DITHER_IGNNOISE = 'ignnoise';
+
+/**
+ * Name of event fired before the camera renders the scene.
+ *
+ * @type {string}
+ * @category Graphics
+ */
+export const EVENT_PRERENDER = 'prerender';
```

Comment:
Do we even need these as constants? What do they give us?

Reply:
I use them internally in the engine, as string typing is error-prone.
but yeah, I don't think it's needed in general.

Reply:
Then at least keep them out of the API ref.

Reply:
we have others that are public and used by the way .. I just followed this as an example :)

<img width="447" alt="Screenshot 2024-11-29 at 15 58 10" src="https://github.com/user-attachments/assets/dcd8c076-dc5d-4dea-b632-5e2ad780873b">

Reply:
removed.